include/class.config.php

```diff
@@ -207,6 +207,7 @@ class OsticketConfig extends Config {
         'agent_avatar' => 'gravatar.mm',
         'ticket_lock' => 2, // Lock on activity
         'max_open_tickets' => 0,
+        'files_req_auth' => 0,
```

Comment:
So I guess that officially retires the idea of sending attachment links for large attachments in emails?